sklearn/compose/_column_transformer.py

```diff
@@ -349,15 +352,11 @@ def named_transformers_(self):
         return Bunch(**{name: trans for name, trans, _
                         in self.transformers_})
 
-    def get_feature_names(self):
-        """Get feature names from all transformers.
-
-        Returns
-        -------
-        feature_names : list of strings
-            Names of the features produced by transform.
+    def _get_feature_names_out(self, get_names):
+        """Private function to be used by get_feature_names_out and
```

Comment:
You might want to add a `# TODO` for a later search and maybe some version number.

Reply:
This should be 0.26.